config/webpack.plugins.js

```diff
@@ -1,24 +1,27 @@
 const webpack = require('webpack');
-const LodashWebpackPlugin = require('lodash-webpack-plugin');
 const WriteFileWebpackPlugin = require('write-file-webpack-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
-const HardSourceWebpackPlugin = require('hard-source-webpack-plugin');
+const path = require('path');
+
+const { ModuleFederationPlugin } = require('webpack').container;
 
 const plugins = [
-    new CleanWebpackPlugin(),
-    new HardSourceWebpackPlugin(),
-    new WriteFileWebpackPlugin(),
-    new webpack.SourceMapDevToolPlugin({
-        test: /\.js/i,
-        exclude: /node_modules/i,
-        filename: `sourcemaps/[name].js.map`
-    }),
-    new LodashWebpackPlugin({
-        currying: true,
-        flattening: true,
-        placeholders: true,
-        paths: true
-    })
+  new ModuleFederationPlugin({
+    name: 'chrome',
+    library: { type: 'var', name: 'chrome' },
+    filename: 'remoteEntry.js',
+    exposes: {
+      './foo': path.resolve(__dirname, '../src/js/chrome.js'),
```

Comment:
Shouldn't this be exposed on `./chrome` path?

Reply:
This does not expose and does not consume anything at this point. It's there just to make sure the build passes. It will be replaced by the config from the hackaton.